README.md

```diff
@@ -231,6 +237,7 @@ Please [discover modm's peripheral drivers for your specific device][discover].
 <td align="center">✗</td>
 <td align="center">✗</td>
 <td align="center">✗</td>
+<td align="center">✗</td>
```

Comment:
This is not 100% true. Ethernet is present for V71 targets (I updated hal matrix generator to accept its name), but I didn't add V71 support so modm-devices doesn't know that. Should I leave it as-is?